### PR TITLE
Demo : Work on the demo #696

### PR DESCRIPTION
**Pull request by cserf**
Demo : Work on the demo #696

### CODE REVIEW

File: doc/source/rucio_demo.rst [hunk 1]
```diff
@@ -0,0 +1,144 @@
+Rucio demo
+==========
+
+Prerequisites
+~~~~~~~~~~~~~
+
+The only prerequesit is to install docker and docker-compose
```
Comment (mlassnig) [inline doc/source/rucio_demo.rst]: prerequisite

(and a missing dot at the end ;-)

File: doc/source/rucio_demo.rst [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+Starting a Rucio demo instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The instructions can be found in : https://github.com/rucio/rucio/tree/master/etc/docker/demo
+
+The bootstrap script creates a Rucio instance with 2 accounts (root and jdoe) and 2 local RSEs (SITE1_DISK, SITE2_DISK) mounted on the /tmp partition. Once everything is ready you can log into the container and start playing around with rucio::
+
+  $ docker exec -i -t demo_rucio_1 /bin/bash
+  [root@3a6d4527e1f6 rucio]# rucio whoami
```
Comment (mlassnig) [inline doc/source/rucio_demo.rst]: blank line to separate the docker command from the rucio command?

File: doc/source/rucio_demo.rst [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+
+Now if you list the rule for this dataset, you will see that there are no files listed on SITE1_DISK::
+
+   [root@3a6d4527e1f6 rucio]# rucio list-rules tests:test.24659.automatix_stream.recon.AOD.917
+   ID                                ACCOUNT    SCOPE:NAME                                       STATE[OK/REPL/STUCK]    RSE_EXPRESSION      COPIES  EXPIRES (UTC)    CREATED (UTC)
+   --------------------------------  ---------  -----------------------------------------------  ----------------------  ----------------  --------  ---------------  -------------------
+   7744c0e0dcce4243b906a2afbc8bc87f  root       tests:test.24659.automatix_stream.recon.AOD.917  OK[0/0/0]               SITE1_DISK               1                   2018-02-19 13:47:06
+
+The information needs to be updated by an other daemon called the judge. To run it once::
```
Comment (mlassnig) [inline doc/source/rucio_demo.rst]: another

File: etc/docker/demo/setup_data.py [hunk 4]
```diff
@@ -1,24 +1,75 @@
...
-# Copyright 2014-2018 CERN for the benefit of the ATLAS collaboration.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# You may not use this file except in compliance with the License.
-# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
-#
-# Authors:
-# - Thomas Beermann, <thomas.beermann@cern.ch>, 2018
+#!/usr/local/bin python
+"""
```
Comment (bari12) [inline etc/docker/demo/setup_data.py]: Are we going for """ or # now? @vingar went for # in his PR. I feel like we are moving in circles there 😆

Comment (mlassnig) [reply etc/docker/demo/setup_data.py]: I'm strongly opinionated towards #

Comment (mlassnig) [reply etc/docker/demo/setup_data.py]: and in any case hardcoding /usr/local/bin/python looks weird, shouldn't this be /usr/bin/env python ?

Comment (tbeerman) [reply etc/docker/demo/setup_data.py]: I'm also in favour of #. I just unitized all headers for the flask files to use #.

Comment (vingar) [reply etc/docker/demo/setup_data.py]: the recommended way is with #. there is an issue about it #??. I'll submit the script to auto generate the header (git log etc) tomorrow. It' helps a lot

Comment (cserf) [reply etc/docker/demo/setup_data.py]: :man_facepalming: 
I'm lost I thought we decided the """

Comment (bari12) [reply etc/docker/demo/setup_data.py]: Let it be decided for all eternity that we use # as the header character.

Comment (vingar) [reply etc/docker/demo/setup_data.py]: see #534

**Review by vingar — APPROVED** [review]

**Pull request merged**